src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -1144,6 +1166,121 @@ export default class SourceFileLinter {
 		}
 	}
 
+	#analyzeNewAndApplySettings(node: ts.NewExpression | ts.CallExpression) {
+		node?.arguments?.filter((arg) => ts.isObjectLiteralExpression(arg))
+			.flatMap((arg) => arg.properties)
+			.forEach((prop) => {
+				if (ts.isPropertyAssignment(prop) &&
+					(ts.isCallExpression(node) ||
+						this.#isPropertyBinding(node, getPropertyNameText(prop.name)))
+				) {
+					if (ts.isObjectLiteralExpression(prop.initializer)) {
+						this.#analyzePropertyBindings(prop.initializer, ["type"]);
+					} else {
+						this.#analyzePropertyStringBindings(prop);
+					}
+				}
+			});
+	}
+
+	#analyzePropertyBindings(node: ts.ObjectLiteralExpression, propNames: string[]) {
+		node?.properties.forEach((prop) => {
+			if (!ts.isPropertyAssignment(prop)) {
+				return;
+			}
+
+			// Get the type property
+			let propertyField;
+			if (ts.isObjectLiteralExpression(prop.initializer)) {
+				propertyField = getPropertyAssignmentsInObjectLiteralExpression(
+					propNames, prop.initializer)[0];
+			} else if (ts.isStringLiteralLike(prop.initializer) &&
+				ts.isIdentifier(prop.name) && propNames.includes(prop.name.text) &&
+				// Whether it's a direct property of the Control
+				// or name collision in property binding
+				!ts.isNewExpression(prop.parent.parent)) {
+				propertyField = prop;
+			}
+
+			if (propertyField) {
+				this.#analyzeBindingPropertyNode(propertyField.initializer);
+			}
+		});
+	}
+
+	#analyzePropertyStringBindings(node: ts.PropertyAssignment) {
+		if (ts.isStringLiteralLike(node.initializer) &&
+			node.initializer.text.startsWith("{") && node.initializer.text.endsWith("}")) {
+			const imports = this.sourceFile.statements
+				.filter((stmnt): stmnt is ts.ImportDeclaration =>
+					stmnt.kind === ts.SyntaxKind.ImportDeclaration)
+				.map((importNode) => {
+					return {
+						moduleName: (importNode.moduleSpecifier as ts.StringLiteral).text,
+						variableName: importNode.importClause?.name?.text,
+					} as RequireDeclaration;
+				});
+
+			const {start: nodePos} = getPositionsForNode({
+				node: node,
+				sourceFile: this.sourceFile,
+				resourcePath: this.resourcePath,
+			});
```

Comment:
traceMap is missing here, so the line/column information might be wrong.

Reply:
Done